From 592da997559dbdfb6cde3f6a83368398be16a3a8 Mon Sep 17 00:00:00 2001
From: Jonathan Moore Liles <j.liles@unix.net>
Date: Wed, 31 Oct 2012 23:27:53 -0700
Subject: [PATCH 0044/1936] s/NTK_FLUID_EXECUTABLE/FLTK_FLUID_EXECUTABLE/

Deal with the fact that the latter apparently hardcoded into cmake.
---
 cmake/FindNTK.cmake | 6 +++---
 src/CMakeLists.txt  | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/cmake/FindNTK.cmake b/cmake/FindNTK.cmake
index d9932deb..8a1fe06c 100644
--- a/cmake/FindNTK.cmake
+++ b/cmake/FindNTK.cmake
@@ -19,14 +19,14 @@ else (NTK_LDFLAGS)
     string(REPLACE "-l" "" NTK_LIBS ${NTK_LIBS})
     string(REPLACE " " "; " NTK_LIBS ${NTK_LIBS})
     #list(APPEND NTK_LIBS ${NTK_LINK_LIBS})
-    find_program (NTK_FLUID_EXECUTABLE ntk-fluid)
-        if (NTK_FLUID_EXECUTABLE)
+    find_program (FLTK_FLUID_EXECUTABLE ntk-fluid)
+        if (FLTK_FLUID_EXECUTABLE)
 #            mark_as_advanced(NTK_CONFIG)
 #            mark_as_advanced(NTK_EXECUTABLE)
 #            mark_as_advanced(NTK_LIBRARIES)
             set(NTK_FOUND TRUE)
             set(NTK_WRAP_UI 1)
-        endif(NTK_FLUID_EXECUTABLE)
+        endif(FLTK_FLUID_EXECUTABLE)
     endif (NTK_CONFIG)
 endif (NTK_LDFLAGS)
 
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 8de9d098..0899ba0e 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -221,7 +221,7 @@ if(NTK_FOUND)
 	mark_as_advanced(FORCE NTK_BASE_LIBRARY)
 	mark_as_advanced(FORCE NTK_CONFIG_SCRIPT)
 	mark_as_advanced(FORCE NTK_DIR)
-	mark_as_advanced(FORCE NTK_FLUID_EXECUTABLE)
+	mark_as_advanced(FORCE FLTK_FLUID_EXECUTABLE)
 	mark_as_advanced(FORCE NTK_FORMS_LIBRARY)
 	mark_as_advanced(FORCE NTK_GL_LIBRARY)
 	mark_as_advanced(FORCE NTK_IMAGES_LIBRARY)
-- 
2.47.0

